Makefile

```diff
@@ -101,5 +101,14 @@ lint-changes:
 lint:
 	golangci-lint run ./...
 
+temp-bindings:
```

Comment:
maybe `typescript-bindings-fixtures`? Not sure why these are 'temporary' if we are comitting them.